docs/en/observability/gcp-dataflow.asciidoc

```diff
@@ -0,0 +1,135 @@
+[[gcp-dataflow]]
+=== GCP Dataflow Templates 
+
+In this tutorial, you'll learn how to ship logs directly from the Google Cloud
+Console with the Dataflow template for analyzing GCP Audit Logs in the Elastic
+Stack.
+
+[discrete]
+==== What you'll learn
+
+You'll learn how to:
+
+- Export GCP audit logs through Pub/Sub topics and subscriptions.
+- Ingest logs using https://cloud.google.com/dataflow[Google Dataflow] and
+view those logs in {kib}.
+
+[discrete]
+==== Before you begin
+
+Create a deployment using our hosted {ess} on {ess-trial}[{ecloud}].
+The deployment includes an {es} cluster for storing and searching your data,
+and {kib} for visualizing and managing your data.
+For more information, see <<spin-up-stack,Spin up the Elastic Stack>>.
+
+[discrete]
+==== Step 1: Install the GCP Integration 
+
+You’ll start with installing the Elastic GCP integration to include prebuilt
+dashboards, ingest node configurations, and other assets that help you get
+the most of the GCP logs you ingest.
+
+First, access *Integrations* in {kib} and search for `gcp`.
+
+image::monitor-gcp-kibana-integrations.png[{kib} integrations]
+
+Next, access the Elastic GCP integration and click
+*Add Google Cloud Platform (GCP)*.
+
+image::monitor-gcp-integration.png[GCP integration]
+
+Then, click *Save integration*.
+
+[NOTE]
+====
+This tutorial assumes the Elastic cluster is already running.
+See below how to get your *Cloud ID* and generate an *API Key* that you'll
+need in the next steps. 
+====
+
+You can find the Cloud ID of your https://cloud.elastic.co/deployments[deployment]
+at its *Overview* page.
+
+image:monitor-gcp-cloud-id.png[Cloud ID]
+
+And you can use {kibana-ref}/api-keys.html#create-api-key[{kib}] to create a
+Base64-encoded API key to authenticate on your deployment.
+
+[IMPORTANT]
+====
+You can optionally restrict the privileges of your API Key, otherwise they'll
+be a point in time snapshot of permissions of the authenticated user.
+For this tutorial the data is written to the `logs-gcp.audit-default` data streams.
+====
+
```

Comment:
Here I'm suggesting some minor edits plus a formatting change that puts a blue box around your description so that all the related (but somewhat ancillary) info is grouped together. I didn't bother copying over the image, but you might need to crop it further to get it to work in the blue box. It'll look something like this:

![image](https://user-images.githubusercontent.com/14206422/138372941-51f12455-82e8-4701-bd48-80a60379abab.png)